docs/examples/solana/payer_annotation.sol

```diff
@@ -0,0 +1,43 @@
+import 'solana';
+
+@program_id("SoLDxXQ9GMoa15i4NavZc61XGkas2aom4aNiWT6KUER")
```

Comment:
This is an example for the docs. I have explained it in `docs/targets/solana.rst`. Should I write more words here too?

Reply:
I don't have a hard opinion on this. Usually it's just easier to just add some comments directly above the relevant LOC instead of trying to explain everything in detail in the doc paragraph. Additionally if someone is browsing our example he'll see the comments and better understand what's going on.